pkg/auth/webapp/cookie.go

```diff
@@ -41,6 +42,7 @@ func NewErrorTokenCookieDef() ErrorTokenCookieDef {
 		AllowScriptAccess: false,
 		SameSite:          http.SameSiteLaxMode,
 		MaxAge:            nil, // Use HTTP session cookie; expires when browser closes
+		IsNonHostOnly:     false,
```

Comment:
No, you do not need to specify false. The whole purpose of choosing a sensible zero value is to allow us to skip specifying false in many places. Please only keep the lines with trues.